src/marqo/tensor_search/tensor_search.py

```diff
@@ -1759,14 +1788,16 @@ def add_prefix_to_queries(queries: List[BulkSearchQueryEntity]) -> List[BulkSear
             scoreModifiers=q.scoreModifiers,
             index=q.index,
             modelAuth=q.modelAuth,
-            text_query_prefix=q.text_query_prefix
+            text_query_prefix=q.text_query_prefix,
+            hybridParameters=q.hybridParameters
```

Comment:
[minor] naming convention: seems we use both camelCase and snake_case extensively :)

Reply:
agreed, in this case we probably want to change all snake case to camel case. Although maybe not in this pr